supertokens_python/recipe/session/utils.py

```diff
@@ -418,4 +463,99 @@ def validate_and_normalise_user_input(
         app_info.framework,
         app_info.mode,
         jwt,
+        invalid_claim_status_code if (invalid_claim_status_code is not None) else 403,
+    )
+
+
+async def get_required_claim_validators(
+    session: SessionContainer,
+    override_global_claim_validators: Optional[
+        Callable[
+            [List[SessionClaimValidator], SessionContainer, Dict[str, Any]],
+            MaybeAwaitable[List[SessionClaimValidator]],
+        ]
+    ],
+    user_context: Dict[str, Any],
+) -> List[SessionClaimValidator]:
+    from .recipe import SessionRecipe
+
+    claim_validators_added_by_other_recipes = (
+        SessionRecipe.get_claim_validators_added_by_other_recipes()
+    )
+    global_claim_validators = await resolve(
+        SessionRecipe.get_instance().recipe_implementation.get_global_claim_validators(
+            session.get_user_id(),
+            claim_validators_added_by_other_recipes,
+            user_context,
+        )
     )
+
+    if override_global_claim_validators is not None:
+        return await resolve(
+            override_global_claim_validators(
+                global_claim_validators, session, user_context
+            )
+        )
+
+    return global_claim_validators
+
+
+async def update_claims_in_payload_if_needed(
+    claim_validators: List[SessionClaimValidator],
+    new_access_token_payload: Dict[str, Any],
+    user_id: str,
+    user_context: Dict[str, Any],
```

Comment:
please make the args ordering of this the same as in nodejs sdk

Reply:
fixed..